File: src/main/src/org/wdt/intellijmanager/utils/OptionUtils.java
package org.wdt.intellijmanager.utils;

import org.apache.commons.cli.Option;

public class OptionUtils {
    public static Option getOption(String command, String longCommand) {
        return getOption(command, longCommand, false);
    }

    public static Option getOption(String command, String longCommand, boolean hasArgs) {
        return Option.builder(command).longOpt(longCommand).hasArg(hasArgs).required(false).build();
    }
}

File: src/main/src/org/wdt/intellijmanager/utils/OptionUtils.kt
package org.wdt.intellijmanager.utils

import org.apache.commons.cli.Option

object OptionUtils {
    @JvmStatic
    fun getOption(command: String?, longCommand: String?): Option {
        return getOption(command, longCommand, false)
    }

    fun getOption(command: String?, longCommand: String?, hasArgs: Boolean): Option {
        return Option.builder(command).longOpt(longCommand).hasArg(hasArgs).required(false).build()
    }
}
